### PR TITLE
Repository name added in response of pullrequests API

### PR DESCRIPTION
**Pull request by adnanraghib1308**
This is the solution for issue #102

### CODE REVIEW

File: controllers/pullRequestsController.js [hunk 1]
```diff
@@ -14,14 +14,17 @@ const getPRdetails = async (req, res) => {
...
         const allAssignees = assignees.map(object => object.login)
         const allLabels = labels.map(object => object.name)
+        const repositoryUrlSplit = repositoryUrl.split('/')
+        const repositoryName = repositoryUrlSplit[repositoryUrlSplit.length - 1]
         allPRs.push({
           title,
           state,
           createdAt,
           updatedAt,
+          repositoryName,
```
Comment (swarajpure) [inline controllers/pullRequestsController.js:27]: `repositoryName` seems too lengthy... What about `repository`? Short and conveys the meaning

**Review by swarajpure — CHANGES_REQUESTED** [review]
Great! Thank you!
Requesting a small change. Please check.

File: controllers/pullRequestsController.js (hunk 1 above)
Comment (adnanraghib1308) [reply controllers/pullRequestsController.js:27]: Sure.

**Comment by adnanraghib1308** [comment]
Its done you can check now.

**Review by swarajpure — APPROVED** [review]

**Pull request merged**